performance/vip-tweaks.php

```diff
@@ -15,8 +16,21 @@ function wpcom_vip_enable_performance_tweaks() {
 	if ( function_exists( 'wpcom_vip_enable_maybe_skip_old_slug_redirect' ) ) {
 		wpcom_vip_enable_maybe_skip_old_slug_redirect();
 	}
+
+	if ( is_admin() ) {
+		// Prevent expensive media library queries, and cache the available months for filtering.
+		add_filter( 'media_library_show_video_playlist', '__return_true' );
```

Comment:
Do we still need to force return true if the default value is already true? https://github.com/WordPress/WordPress/blob/39bff93b6b274a873158a362725b1338ab1400e5/wp-includes/media.php#L4323

Reply:
Nice find, looks like we can remove these now since WP 4.8 made it the default. These filters predate this PR, I just moved them up a few lines.